### PR TITLE
.github: don't depend on COCKPIT_REPO_STAMP target

### PR DESCRIPTION
**Pull request by jelly**
Instead of relying on COCKPIT_REPO_STAMP which changed in
fca470b5135228fcd8865 use the existing make target.

### CODE REVIEW

**Comment by jelly** [comment]
@allisonkarlitskaya do I need to make my PR from `origin`?

File: .github/workflows/reposchutz.yml [hunk 1]
```diff
@@ -56,7 +56,7 @@ jobs:
       - name: Check for node_modules availability and package.json consistency
         run: |
           # Make tools/node-modules available
-          make pkg/lib/cockpit.js
+          make tools/node_modules
```
Comment (allisonkarlitskaya) [inline .github/workflows/reposchutz.yml:59]: I think you meant `tools/node-modules`?

**Review by allisonkarlitskaya — CHANGES_REQUESTED** [review]

**Comment by allisonkarlitskaya** [comment]
> @allisonkarlitskaya do I need to make my PR from `origin`?

No.  But you need the label set.  I just created it.

File: .github/workflows/reposchutz.yml (hunk 1 above)
Comment (jelly) [reply .github/workflows/reposchutz.yml]: Yes, danke

**Comment by jelly** [comment]
> > @allisonkarlitskaya do I need to make my PR from `origin`?
> 
> No. But you need the label set. I just created it.

Aha, but then it still fails as it takes the current workflow from main?

**Comment by allisonkarlitskaya** [comment]
> > > @allisonkarlitskaya do I need to make my PR from `origin`?
> > 
> > 
> > No. But you need the label set. I just created it.
> 
> Aha, but then it still fails as it takes the current workflow from main?



> > > @allisonkarlitskaya do I need to make my PR from `origin`?
> > 
> > 
> > No. But you need the label set. I just created it.
> 
> Aha, but then it still fails as it takes the current workflow from main?

Correct.  I propose we merge this and then test with another PR.

**Review by allisonkarlitskaya — APPROVED** [review]

**Pull request merged**